src/corecel/math/detail/MathImpl.hh

```diff
@@ -55,7 +55,7 @@ inline void sincospi(T x, T* sptr, T* cptr)
     }
     else
     {
-        x *= m_pi;
+        x *= static_cast<T>(m_pi);
```

Comment:
Not relate to this change but this template function could use a limiter (concept in C++) as it is valid only for floating point types.

Reply:
The public one that calls this has a limiter. The detail class won't cause any ambiguity.